src/main/java/BuyStockUseCase/BuyInputRequest.java

```diff
@@ -8,7 +8,7 @@ public class BuyInputRequest {
      * Request model class, carries user input to be passed to the BuyUseCaseInteractor.
      */
 
-    String symbol;
+    private String symbol;
     int buy_quantity;
```

Comment:
should all be private

Reply:
idk how I missed the other 2, fixed it now